KinConnections/resources/email_wrapper.py

```diff
@@ -0,0 +1,96 @@
+# email_wrapper.py - KinConnections
+# Serves as wrapper for sending connection emails
+
+# for environment variables
+import os
+from dotenv import load_dotenv
+# for email engine
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.utils import formataddr
+# for rich html email
+from dominate import document
+from dominate.tags import *
+# flask
+from flask import Blueprint, Response, request, render_template, session, redirect, url_for
+from flask_restful import Resource
+
+from .db_wrapper import *
+
+
+# access credentials from .env
+load_dotenv(dotenv_path="../.env")
+
+kc_email = os.getenv("KC_EMAIL")
+kc_email_password = os.getenv("KC_EMAIL_PASSWORD")
+kc_email_server = os.getenv("KC_EMAIL_SERVER")
+kc_email_port = int(os.getenv("KC_EMAIL_PORT"))
+
+class Email(Resource):
+    def get(self):
```

Comment:
We should probably just get rid of this. Since a non-existent URL call to the server will just return a `404` by default.